### PR TITLE
Informative Comment Update: python.BQ_explore_data.ipynb

### PR DESCRIPTION
**Pull request by akshaykumarpatil-tudip**
- Added the informative comments with all description in the solution notebook.
- Tested the notebook with changes and is working fine.

### CODE REVIEW

File: courses/machine_learning/deepdive2/launching_into_ml/solutions/python.BQ_explore_data.ipynb [hunk 1]
```diff
@@ -41,6 +41,7 @@
    "metadata": {},
```
Comment (gstripling) [inline courses/machine_learning/deepdive2/launching_into_ml/solutions/python.BQ_explore_data.ipynb:41]: <p>I suggest we delete line(s) 2, 3, and 4.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1266/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/launching_into_ml/solutions/python.BQ_explore_data.ipynb:41]: <p>Remove line 7, I don't know why I left that in.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1266/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/launching_into_ml/solutions/python.BQ_explore_data.ipynb:41]: <p>Line 1, "value".  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1266/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/launching_into_ml/solutions/python.BQ_explore_data.ipynb:41]: <p>Line 1, "plots".</p><p>Line 2, "finds"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1266/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (akshaykumarpatil-tudip) [reply courses/machine_learning/deepdive2/launching_into_ml/solutions/python.BQ_explore_data.ipynb:41]: > Line 1, "plots".
> 
> Line 2, "finds"
> 
> 
> _Reply via [ReviewNB](https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1266/discussion/)_

Thanks Gwendolyn,
Updated this and all above suggested changes.

Comment (akshaykumarpatil-tudip) [reply courses/machine_learning/deepdive2/launching_into_ml/solutions/python.BQ_explore_data.ipynb:41]: Updated the change.

[the remark this answers is not in the document]
Comment (akshaykumarpatil-tudip) [reply courses/machine_learning/deepdive2/launching_into_ml/solutions/python.BQ_explore_data.ipynb:41]: Updated the change.

[the remark this answers is not in the document]
Comment (akshaykumarpatil-tudip) [reply courses/machine_learning/deepdive2/launching_into_ml/solutions/python.BQ_explore_data.ipynb:41]: Updated the change.

Comment (akshaykumarpatil-tudip) [reply courses/machine_learning/deepdive2/launching_into_ml/solutions/python.BQ_explore_data.ipynb:41]: Updated the change.

Comment (akshaykumarpatil-tudip) [reply courses/machine_learning/deepdive2/launching_into_ml/solutions/python.BQ_explore_data.ipynb:41]: Remove the lines as required.

**Comment by akshaykumarpatil-tudip** [comment]
Thanks @gstripling 

Updated all the suggested changes. Feel free to merge now.

**Pull request merged**